programmers/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,17 @@
+function solution(numbers) {
+    
+    const NUM_LEN = numbers.length;
+    let answer = [];
+    let sortAnswer = [];
```

Comment:
```suggestion
    const answer = [];
    const sortAnswer = [];
```
배열이나 오브젝트같은 경우는 객체이기 때문에 const로 선언해도 객체 안의 값은 변화시킬 수 있어요
하지만 객체 자체를 재정의하지 못하기 때문에 안정성이 더 높아져서
배열이나 오브젝트는 const로 선언해줍니다.